### DISCUSSION TITLE
Hi, could you explain me how multilanguage_domain preference works?

### DISCUSSION THREAD

**Discussion by Jimmi08** [e107inc/e107#5144] [Q&A Core]
If I have 2 domains, on one I want to display one language, on the other one second or both.
But only one e107 installation.
Is it possible? 
 
Help is only "Domain determines the site's language. Enter domain without the 'www.' ", But how to set domain itself?

**Comment by LaocheXe** [comment]
An if statement should do it.

$_SERVER['HTTP_HOST']; is a way, but lacks security.

Wordpress uses 
$urlparts = wp_parse_url(home_url());
$domain = $urlparts['host'];

Drupal - 
\Drupal::request()->getHost();

so the question is what does e107 use?

**Comment by Alex-e107nl** [comment]
If I remember correctly, you must have a domain pointer on your domain of the domain that you want to link to the language you want, simply directing an external domain does not work. If you can do it with subdomains, it will also work with additional domains. But like I said, it's been a really long time since Ive created a multi-language website that way, nowadays just the pull field in the menu...

**Reply by Jimmi08** [reply]
I am waiting to finish the registration process, they offer "masking redirection" - it means that one domain displays the content of the second domain. I will see if e107sk.com can detect the different domains and display different languagese.

**Comment by Jimmi08** [comment]
It works. Very nice, I don't think this is the feature of any other CMS from the core. 
Now I need to wrap up my head about content.  Yeah, I know the theory, I tried create table - I just somehow supposed that it copies English content and I will only replace text (f.e. Sitelinks),  Nope.

**Reply by Alex-e107nl** [reply]
You need the multilan plugin.... it works! But english has to be the main language...
